syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -347,6 +349,8 @@ def add(self, shares: dict, other):
                 crypto_provider=self.crypto_provider,
                 **no_wrap,
             ).child
+        elif isinstance(other, AdditiveSharingTensor):
```

Comment:
This is not supposed to happen: the decorator makes sure that the AST other is replaced by his .child

So if you still have an AST it means that you've operated something like [syfttype]>AST>... with the self AST

What is the usecase for this?